qt/scientific_interfaces/Muon/ALCInterface.cpp

```diff
@@ -285,5 +287,50 @@ void ALCInterface::importPeakData(const std::string &workspaceName) {
   }
 }
 
+void ALCInterface::externalPlotRequested() {
+  MatrixWorkspace_sptr data;
+  std::vector<QHash<QString, QVariant>> kwargs;
```

Comment:
I assume each element of the kwargs vector corresponds to a different workspace?

Reply:
I have renamed this to a better name